server/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const fetch = require('node-fetch');
+const path = require("path")
+const bodyParser = require("body-parser")
+const searchFunctions = require("./functions")
+
+const port = 3000;
+app.listen(port, () => {
+    console.log(`Server listening on ${port}`)
+})
```

Comment:
I would move this to a separate file such as index.js for the sole purpose of just running the server. This isn't an issue at the moment but can cause problems when doing testing.